src/Compatibility/Microsoft.DotNet.PackageValidation/Helpers.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.ApiCompatibility.Abstractions;
+using NuGet.ContentModel;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.PackageValidation
+{
+    internal static class Helpers
```

Comment:
I would name this Extensions or something like that.

Reply:
i will do the name change in other open pr